tree/20_cfe_basics/dry_run_lib.cf

```diff
@@ -34,6 +54,11 @@
 # WARNING: * promise type specific dry-run is NOT considered by action bodies, and will be overriden
 # WARNING:   when using any action body.
 # WARNING:
+#
+# @parameter mode   The dry-run mode, "true" to enable dry-run-mode and "false" to disable it
+# @agent_requirements "capabilities": [ "defaultbody" ] || "agent_version"" >= 3.9
```

Comment:
I know, but we will need it for this. We need to choose the syntax and implement it in list-compatible-inputs.